matlab_simulation/05-estimation/kalman_filter/kf_example3.m

```diff
@@ -0,0 +1,105 @@
+% Kalman Filter Example 2: Multirate 2D Omnidirectional AUV example
+% 'Estimation 1' topic slide 77
+% State : position and velocity in north, east directions
+% Motion : movement in north and east directions
+% Measurement : measure position(10Hz) and velocity(100 Hz) in north and 
+%               east directions
+% Controller : Thrust in north and east directions 
+
+clear all
+clc
+close all
+
+% Example number
+example = 3;
+
+% Create AVI object
+makemovie = 1;
+if(makemovie)
+    vidObj = VideoWriter('multiratekf.avi');
+    vidObj.Quality = 100;
+    vidObj.FrameRate = 8;
+    open(vidObj);
+end
+
+% Discrete time step
+dt = 0.01;
+
+% Prior
+mu = zeros(4,1); % mean (mu)
+S = 0.01*eye(4);% covariance (Sigma)
+
```

Comment:
Same thing here as in Kalman Filter example II, you could use the n from the motion model (line 33 in this file) to define the size of prior.